pysal_2-1_logo/create_pysal_logo.py

```diff
@@ -0,0 +1,362 @@
+""" Create the PySAL 2.1.0 logo with TeX/TikZ, then create
+    favicons at specified resolutions with ImageTricks.
+    The original logo design was based on Figure 1 from Rey and Anselin (2007).
+    
+    Rey, S. J. and Anselin, L. (2007). PySAL: A python library of
+        spatial analytical methods. The Review of Regional Studies, 37(1):5–27.
+
+Examples from the command line:
+    (without child node text)
+    $ python create_pysal_logo.py
+    (with child node text)
+    $ python create_pysal_logo.py True
+
+Note 1:
+    Changing child colors can be done in `_7_NODE_COLORS`, but use
+    caution as some colors don't blend well when combining schema as
+    is done here. For example, instead of 'gray' for the root node
+    color use 'rgb:black,1.25;white,1' (the default argument value for
+    `concept_color` in `create_logo()`). Also, note that RBG colors
+    do not (at the time of the writing) blend well with CMY colors.
+    See the following:
+    https://ipfs-sec.stackexchange.cloudflare-ipfs.com/tex/A/question/24434.html
+    https://tex.stackexchange.com/questions/48662/tikz-or-xcolor-lighten-color
+    https://tex.stackexchange.com/questions/308853/why-does-xcolor-lighten-a-color-when-mixed
+
+Requirements:
+    Python 3.6+
+    LuaTeX, Version 1.10.0 (TeX Live 2019)
+    M+ fonts
+        info -- https://mplus-fonts.osdn.jp/about-en.html
+        download -- https://osdn.net/projects/mplus-fonts/releases/<RELEASE>
+            The files created with the initial push for this file were run
+            on release <62344>. The current release for download is <p14454>.
+    ImageTricks (for favicon creation)
+        https://www.belightsoft.com/products/imagetricks/
+
+Authors:
+    Serge Rey,
+    Luc Anselin,
+    James Gaboardi <jgaboardi@gmail.com>,
+    Wei Kang,
+    Eli Knaap,
+    others?
+
+File creation date:
+    2019-12
+"""
+
+from ast import literal_eval
+import subprocess
+import sys
+import time
+
+# set this for PySAL meta package release
+__version__ = "2.1.0"
+
+# Predefined file name, node colors, and node text (if desired)
+OUT_FILE_BASE = "pysal_logo"
+_7_NODE_COLORS = [
+    "violet",
+    "black!5!red",
+    "blue",
+    "black!50!green",
+    "brown",
+    "black!40!red",
+    "orange",
+]
+_7_NODE_TEXT = [r"theta", r"gamma", r"tau", r"lambda", r"alpha", r"W", r"rho"]
+_7_NODE_TEXT = [r"$\" + t + "$" for t in _7_NODE_TEXT]
```

Comment:
I ran into syntax error (line 69) when running `python create_pysal_logo.py True` on my mac (python 3.7). Changing lines 68 and 69 to 
```
_7_NODE_TEXT = [r"$\theta$", r"$\gamma$", r"$\tau$", 
                r"$\lambda$", r"$\alpha$", r"$W$", r"$\rho$"]
```
resolves the issue and properly renders latex symbols as well as regular symbol `W`.